blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/TokenDiscoveryIO.java

```diff
@@ -0,0 +1,12 @@
+package com.rackspacecloud.blueflood.io;
+
+
+import com.rackspacecloud.blueflood.types.Token;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface TokenDiscoveryIO  extends BaseDiscoveryIO {
```

Comment:
Lets discuss about the  "aggregated" method and create a story. For now, I am going to keep it.